cmd/wasm-client/permissions.go

```diff
@@ -1,24 +1,76 @@
 package main
 
 import (
+	"net"
+
 	faraday "github.com/lightninglabs/faraday/frdrpcserver/perms"
 	loopd "github.com/lightninglabs/loop/loopd/perms"
 	poold "github.com/lightninglabs/pool/perms"
 	"github.com/lightningnetwork/lnd"
+	"github.com/lightningnetwork/lnd/autopilot"
+	"github.com/lightningnetwork/lnd/chainreg"
```

Comment:
Hmmm, since the wasm client's perms list should always include all perms, perhaps I should just purely copy-paste all LND subserver's perms instead of doing it like this.

Reply:
Hmm... I think that makes it easier for us to forget and not update the permissions. I'm fine with this solution. Maybe we could copy/paste the `PermissionManager` from lit (see my comment in the PR) here and use it, but perhaps that's overkill. Seems pretty straightforward here.

Reply:
ok cool, yeah agreed that it is ok to keep as is here for now since it is only a single function here